Gemfile

```diff
@@ -28,8 +28,9 @@ gem "uglifier", ">= 1.3.0", require: false
 # Explicitly avoid 1.x that doesn't support Ruby 2.4+
 gem "json", ">= 2.0.0"
 
-gem "rubocop", ">= 0.47", require: false
+gem "rubocop", "~> 0.71.0", require: false
```

Comment:
rails/rails repository is maintained by Gemfile.lock. Is this change meaningful?